### PR TITLE
LRQA-73727 Add Poshi Standalone docs

### PR DESCRIPTION
**Pull request by nikki-pru**
https://issues.liferay.com/browse/LRQA-73727

Copied markdown files to [Confluence](https://liferay.atlassian.net/wiki/spaces/~682401262/pages/2002157571/Poshi) to review content and flow.

### CODE REVIEW

File: docs/poshi/latest/en/getting-started/creating-a-path-file.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  1. Type in the search term, 'Pumpkin'.
+
+  1. Press the enter key to enter the search term.
+
+  1. Assert that there are no search results.
+
+  1. Assert that a message appears saying there are no articles available.
+
+## Building the path file.
```
Comment (jrhoun) [inline docs/poshi/latest/en/getting-started/creating-a-path-file.md:19]: @nikki-pru what's the difference between line 19 and line 35?

Comment (jrhoun) [reply docs/poshi/latest/en/getting-started/creating-a-path-file.md:19]: Both say, "Building the Path File"

Comment (nikki-pru) [reply docs/poshi/latest/en/getting-started/creating-a-path-file.md]: Sending fix in https://github.com/jrhoun/liferay-learn/pull/1431/commits/53f218c8979b056b0d49d4017be034fdd92a6db9

**Comment by jrhoun** [comment]
Merged. Thanks!

**Pull request closed without merging**